App.js

```diff
@@ -1,26 +1,131 @@
+import React, { useState } from "react";
+import * as Haptics from "expo-haptics";
+import styled from "styled-components/native";
+import { StatusBar, Alert } from "react-native";
+import { Fetch } from "./components/Fetch";
+import { Header } from "./components/Header";
+import { Footer } from "./components/Footer";
+import { Lottie } from "./components/Lottie";
+import { RandomBeer } from "./components/RandomBeer";
+import { StyledButton } from "./components/StyledButton";
+import { StyledButtonReverse } from "./components/StyledButtonReverse";
 
 const Container = styled.View`
   flex: 1;
-  background-color: papayawhip;
-  justify-content: center;
+  background-color: #fcfaf1;
+  align-items: center;
+`;
+const Wrapper = styled.View`
+  flex: 1;
   align-items: center;
-`
+  justify-content: center;
+`;
+
+const RowWrapper = styled.View`
+  flex-direction: row;
+`;
 
-const Title = styled.Text`
-  font-size: 24px;
-  color: palevioletred;
-`
+const handleOnclick = (beerCount, setBeerCount, setBeer, setPress) => {
+  if (beerCount < 5) {
+    setPress(true);
+    Haptics.impactAsync(Haptics.ImpactFeedbackStyle.Medium);
+    setBeerCount(beerCount + 1);
+  } else {
+    Haptics.impactAsync(Haptics.ImpactFeedbackStyle.Medium);
+    Alert.alert(
+      "Hey there!",
+      "Seems like you're checking out beers fast, don't forget to drink some water 🥤🥤🥤",
+      [
+        {
+          text: "You got it boss! 👍",
+          onPress: () => {
+            Haptics.impactAsync(Haptics.ImpactFeedbackStyle.Medium);
+            setBeerCount(0);
+            setBeer();
+          },
+        },
+      ]
+    );
+  }
+};
 
 const App = () => {
+  const [press, setPress] = useState(false);
+  const [beer, setBeer] = useState();
+  let [beerCount, setBeerCount] = useState(0);
+
   return (
     <Container>
-      <Title>This is your cool app!</Title>
-      <Title>Go to App.js and start coding</Title>
-      <Title>💅💅💅</Title>
+      <StatusBar barStyle="dark-content" hidden={false} />
+      <Header
+        title="🍻 IPA from API 🍻"
+        titleTwo={`The #1 app for beer lovers`}
+      />
+      <Wrapper>
+        {!beer && <Lottie />}
+        <RowWrapper>
+          {beer && (
+            <StyledButtonReverse
+              width="80px"
+              buttonText="Back"
+              pressValue={() => {
+                setBeer();
+                Haptics.impactAsync(Haptics.ImpactFeedbackStyle.Medium);
+              }}
+            />
+          )}
+          <StyledButton
+            buttonText={!beer ? "Time for a 🍺?" : "And another one..."}
+            pressValue={() =>
+              handleOnclick(beerCount, setBeerCount, setBeer, setPress)
+            }
+            fontColor={`#a86913`}
+          />
+        </RowWrapper>
+        {press && <Fetch setBeer={setBeer} />}
+        {beer && <RandomBeer press={press} beer={beer} setPress={setPress} />}
+      </Wrapper>
+      {!beer && (
+        <Footer
+          name={"Christina Persson"}
```

Comment:
you don't really need the curlies around the string here. 
`name="Christina Persson" `
should be enough